GenerateCLP/Testing/PointExample1/PointExample1.xml

```diff
@@ -41,5 +41,12 @@
       <description>Multi Float</description>
       <label>MultiFloat</label>
     </float>
+    <string>
+      <name>simplestring</name>
+      <longflag>--string</longflag>
+      <description>Simple string to test that the escaping by Generate CLP</description>
+      <label>Simple String</label>
+      <default>"This is a test for the escaping"</default>
```

Comment:
Adding test for this would also be good:

```
<default><![CDATA[This doublequote " and this quoted "word" should be escaped]]></default>
```

To lean about CDATA, see https://en.wikipedia.org/wiki/CDATA